components/collective-page/sections/Updates.js

```diff
@@ -171,7 +171,11 @@ class SectionUpdates extends React.PureComponent {
                       </P>
                     </Link>
                     {update.userCanSeeUpdate ? (
-                      <HTMLContent content={update.summary} />
+                      <Container>
+                        <HTMLContent
+                          content={`${update.summary} <a href="${collective.slug}/updates/${update.slug}"><small>Read More</small></a>`}
```

Comment:
There are some issues with this:
1. It's not internationalized . You can re-use the existing string for this one:
```jsx
<FormattedMessage id="ContributeCard.ReadMore" defaultMessage="Read more" />
```
2. It needs to be a React Component, not HTML that we pass through `content` (also to avoid XSS injections)

Reply:
@Betree : Thanks. I've updated the code to use a react component and the `FormattedMessage` but I don't know exactly how we can show both the link and the HTML content on the same line using this method; which is the reason why I used the previous method. :smile:

Reply:
What about:

```jsx
<Container>
  <HTMLContent display="inline" />
  <StyledLink as={Link} display="inline" />
</Container>
```

Reply:
@Betree : I tried this but that doesn't seem to work. The thing here is `HTMLContent` returns a `div` element, unless we pass in the Read More link with that content it might be impossible to show the two elements as a contiguous block of text. Because that would mean one element shows on top of the other element right? Unless we do some weird element overlapping logic :thinking:

Reply:
I just tried in the console, it seems to work. You'll have to remove the `<p>` from inside the link

![image](https://user-images.githubusercontent.com/1556356/89013926-0b904680-d315-11ea-9183-e0c01125e99f.png)

Reply:
@Betree : Ah, yes you are right. Very interesting, didn't know that was possible. Thanks. :smile: